kernel_bench/main.cc

```diff
@@ -133,7 +137,7 @@ KernelBenchApp::KernelBenchApp(int argc, char **argv)
 
   scratch_buff.reserve(nb_workers);
   for (i = 0; i < nb_workers; i++) {
-    scratch_buff.emplace_back(graph.scratch_bytes_per_task, 0);
+    scratch_buff.emplace_back(graph.scratch_bytes_per_task*2, 0);
```

Comment:
I'm a little confused about this change; won't this result in the kernel failing because the number of scratch bytes don't match? If this is for alignment then I'm guessing we're can probably get away without it.

Reply:
ops, I forget to take off the *2. I added it to increase the memory buffer size, so that I can test different kinds of alignment.